framework/resources/Functional/table_function/positive/drill-5166_npe_with_table_function_group_select_plan_check.e

```diff
@@ -0,0 +1 @@
+"pop" : "union-exchange"
```

Comment:
For explain plan queries, we look for the text plan and not the JSON part. Please update accordingly.

Reply:
Also, please seperate out the data and plan verification tests - use a different extension (.plan) and use an additional JSON test def file. The verification type must be changed to regex.